File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution(object):
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        cnt = 0
        for i, c in enumerate(reversed(s)):
            if cnt == 0 and c == " ":
                continue
            elif c == " ":
                return cnt
            else:
                cnt +=1
        return cnt
        

File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        boolean isWord = false;
        int cnt = 0;
        for(int i = s.length()-1; i>=0 ; i--){
            if(isWord == false && s.charAt(i) == ' '){
                continue;
            }else if(isWord == true && s.charAt(i) == ' '){
                return cnt;
            }else{
                isWord = true;
                cnt++;
            }
        }
        return cnt;
    }
}